.github/filters.yml

```diff
@@ -66,7 +66,7 @@ blast_blastn:
   - software/blast/blastn/**
   - tests/software/blast/blastn/**
 
-bowtie:
+bowtie_build:
```

Comment:
Does this work now?

Reply:
Ah no ... sadly it didn't. Tried around a bit and forgot about this change. But actually I think the naming makes more sense this way.
The only way around the bowtie thing is to call the tool different, so not `build`, maybe `build-index` or something

Reply:
It was bowtie_index before, a git blame shows @drpatelh changed it to the proper name, so maybe he can weigh in.